.github/GitReleaseMaanger/.templates/default/issue-note.sbn

```diff
@@ -0,0 +1,6 @@
+{{
+    if issue_label == "Bug"
+}}- Fix - {{ issue.title }} - see [#{{ issue.number }}]({{ issue.html_url }}).
+{{  else
+}}- {{ issue.title }} - see [#{{ issue.number }}]({{ issue.html_url }}).
+{{  end -}}
```

Comment:
Unlike the other template files, this one does not have a new line at the end.

Was that on purpose?

Reply:
No, this was as a result of the refactoring that I did. Let me add that back in.